src/main.cpp

```diff
@@ -2,11 +2,16 @@
 
 #include "Client.h"
 #include "Server.h"
+#include "Log.h"
 
 
 int main() {
     std::cout << "---------- Start ----------" << std::endl;
 
+    Log log;
+    std::cout << log.setLevel(Log::LogLevel::Info) << "test" << std::endl;
```

Comment:
Don't want to deal with std::cout. Make Log self-sufficient: 
`log.setLevel(...) << ...`